Interface.md

```diff
@@ -42,5 +42,13 @@ services through the exploitation of standard protocols and the harmonizing of s
 | **AUTH\#1** |  Authentication interfaces should support the [OpenID Connect](https://openid.net/developers/how-connect-works/) Protocol. |
```

Comment:
I think OIDC is used for user-based authorisation for apps. When I've set it up, it's about configuring centralised single-sign-on for an organisation.

I think for **interfaces**, we should be recommending something like what either NASA do with [Earthdata](https://earthaccess.readthedocs.io/en/stable/howto/authenticate/) or the [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/docs/concepts/sas/) folks do.

To be more specific, these two implementations support programmatic access, i.e., I can write software to sign urls or to provide authentication headers in requests that provide the authorisation to access data.

Reply:
Ah, ok, so I'm misunderstanding. Earthdata use this kind of Bearer header token methodlology you're talking about.

So, I withdraw this comment.